config/examples/dot_product.yaml

```diff
@@ -0,0 +1,62 @@
+---
+type: config
+log_level: DEBUG
+cpu_affinity: [0,1,2,3]
```

Comment:
Some simple comments where would be good in case the reader is going out of order and just looking at the file.